jupyterlab_git/git.py

```diff
@@ -69,6 +98,8 @@ def __init__(self, contents_manager):
         self.contents_manager = contents_manager
         self.root_dir = os.path.expanduser(contents_manager.root_dir)
 
+        self.remote = GitRemote()
```

Comment:
Git repos can (and usually do) have multiple remotes so this seems like a 1:N relationship. WDYT think about making this a list?

Reply:
I will refractor the code. The idea was to get a API close to the git command syntax. But this obviously give the impression some information are store in a `remote` attribute.